debian/control

```diff
@@ -0,0 +1,22 @@
+Source: python-nidaqmx
+Section: python
+Priority: optional
+Maintainer: "Cedric Boudinet" <boudinpg@gmail.com>
+Build-Depends:
+ python3-all,
+ dh-python,
+ pybuild-plugin-pyproject,
+ python3-deprecation,
+ python3-pytest-mock,
+ python3-pykka,
+ python3-tzlocal,
```

Comment:
It looks like Debian stable still has tzlocal 4.x: https://packages.debian.org/en/bookworm/python3-tzlocal

tzlocal broke compatibility in versions 4.0 and 5.0 ( https://pypi.org/project/tzlocal/ ) and we haven't tested nidaqmx with tzlocal 4.x. Do the nidaqmx tests pass with this version?